Fixing grammar in the "Update Introduction to Obsidian Sync" article (#331)
obsidianmd/obsidian-docs: en/Obsidian Sync/Introduction to Obsidian Sync.md

Learn more about Obsidian Sync:

> - If you are a new Obsidian Sync user, learn how to [[set up Obsidian Sync]].

Reviewer: ```suggestion
- If you're a new Obsidian Sync user, learn how to [[set up Obsidian Sync]].
```

We're using contractions for the Obsidian docs.

Author: @marcusolsson thanks for the feedback regarding the writing style on docs! Applied!
